src/Masking.Serilog/ByMasking/MaskingOptions.cs

```diff
@@ -34,5 +34,10 @@ public class MaskingOptions
         /// A flag to specify whether static properties should be included when Destructuring the object.
         /// </summary>
         public bool ExcludeStaticProperties { get; set; } = false;
+        
+        /// <summary>
+        /// List of namespaces for which masking will be ignored.
+        /// </summary>
+        public List<string> Namespaces { get; } = new List<string>();
```

Comment:
What if we call it `IgnoredNamespaces` to make it more clear?

Reply:
Makes perfect sense. Will change this as suggested,